docs/website/docs/getting-started/features.md

```diff
@@ -3,37 +3,30 @@ title: Features
 sidebar_position: 1
 ---
 
-# Features provided by odo
+# Features of odo
```

Comment:
I won't make this change as `odo concepts` title is `Concepts of odo`. If the title was `odo Features` it would look odd as every other title is capitalized and would look out of place, see image:

![Screenshot from 2022-01-24 08-12-40](https://user-images.githubusercontent.com/6422176/150789028-c3d8c35d-5060-4fed-8208-64b7b461cdbf.png)

Reply:
Oh, I didn't realize it is `Concepts of odo` now. In that case, I agree with you, it should be `Features of odo`.